.changeset/modern-queens-complain.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-pantheon-decoupled-kit": patch
```

Comment:
Changed this to a patch. Not sure it's worth a major version for the create- package itself. This does bring up an interesting problem: we'll need to figure out how to maintain versioning of the templates vs the package.